app/views/check-your-answers.html

```diff
@@ -0,0 +1,93 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Check your answers
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "javascript: history.go(-1)"
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <h1 class="govuk-heading-xl">Check your answers</h1>
+
+      <h2 class="govuk-heading-m">Confirm your postal address</h2>
+
+      <p class="govuk-body">
+        This is where we will send your blue badge.
+      </p>
+
+      {{ govukSummaryList({
+        rows: [
+          {
+            key: {
+              text: "Name"
+            },
+            value: {
+              html: "Stephen Gill"
+            },
+            actions: {
+              items: [
+                {
+                  href: "/#",
+                  text: "Change",
+                  visuallyHiddenText: "name"
+                }
+              ]
+            }
+          },
+          {
+            key: {
+              text: "Date of birth"
+            },
+            value: {
+              html: "11 June 1982"
+            },
+            actions: {
+              items: [
+                {
+                  href: "/#",
```

Comment:
If you mean this to not take the user to another page I think you need
```suggestion
                  href: "#",
```